### DISCUSSION TITLE
"Text pushing" behaviour when Zoomed in

### DISCUSSION THREAD

**Discussion by Glint-Eye** [vslinko/obsidian-zoom#76] [Archive]
First, thx for the great plugin.

I'd like to point to the imho "strange" way text is being pushed around, whilst beeing zoomed in. Once your list reached the middle of the workspace, further text is always beeing pushed up, so that half the workspace is basically always empty. At the same time editing items while zoomed into a heading or list with lots of items that fills the whole screen... whenever a new item is beeing added, the new line is beeing pushed up all the way to the top so you have to scroll up everytime you enter a new line.

It is very irritating and i'd love the plugin to optionally just push text normally like in the zoomed out state.

Cheers :)

**Comment by vslinko** [comment]
@Glint-Eye Hi! Thank you for the feedback!

Could you please make a screencast with this problem?

**Reply by vslinko** [reply]
Ow, found an issue https://github.com/vslinko/obsidian-zoom/issues/77 :)
